llama_index/llms/litellm_utils.py

```diff
@@ -170,6 +175,15 @@ def from_openai_message_dict(message_dict: dict) -> ChatMessage:
     return ChatMessage(role=role, content=content, additional_kwargs=additional_kwargs)
 
 
+def from_litellm_message(message: Message) -> ChatMessage:
+    """Convert litellm.utils.Message instance to generic message."""
+    role = message.get("role")
+    # NOTE: Azure OpenAI returns function calling messages without a content key
+    content = message.get("content", None)
+
+    return ChatMessage(role=role, content=content)
```

Comment:
basically `additional_kwargs` got removed from litellm chatmsg compared to before?

Reply:
yes it seems to be the only change affecting my use case (python3.10, macos, openai==1.1.1, litellm==1.1.1, llama-index==0.9.3.post1). 

tested this update on several rag based chat apps utilizing aws, openai, palm, azure llm's with no errors. also tested the latest llama-index updates as extractor pipelines with several retrieval response modes. everything seems to be fine after this update.